catalog/core/catalog-core-localstorageprovider/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -18,7 +18,7 @@
     <OCD name="Content File System Storage Provider"
          id="org.codice.ddf.catalog.content.impl.FileSystemStorageProvider">
         <AD
-                description="Specifies the directory to use for the content repository. A shutdown of the server is necessary for this property to take effect. If a filepath is provided with directories that don't exist, File System Provider will attempt to create them. Out of the box (without configuration), the content repository is <INSTALL_DIR>/content/store."
```

Comment:
it was an issue with generating the tables. It called it "improperly formatted xml"